### PR TITLE
[QUEST] There Exists No Honor Among Birds

### PR DESCRIPTION
**Pull request by DantestyleXD**
**Changes proposed:**
-  Improved waypoints for enemy NPCs
-  Add aura for main NPCs

**Target branch(es):** 
1.x

**Issues addressed:** 
No issues opened on this branch but close issue on link (https://github.com/AzerothShard/AzerothShard-Issues-ita/issues/89)
With this, the quest is fixed on this branch too.

**Tests performed:**
Start quest, use spell and test the quest. Work.

**Known issues and TODO list:**
Known issue:
 - When enemy NPCs lose aggro on the main NPCs, they will attack player.

TODO:
 - Reset enemy NPCs when lose aggro on the main NPCs.

----------------------------
Server: Local (Repo AzerothCore)
Last commit hash tested: 17061cbbb481122317a8a54aef508f3eca475a0b
O.S.: Windows 10
Compiler: MS Visual Studio 2013

### CODE REVIEW

**Comment by Yehonal** [comment]
Please could you move your sql under new pending folders? 
You also need to remove the alter table header, we will add it automatically and move your sql via our script after PR merging

File: data/sql/dbfix/2016_08_23_00.sql [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+(@PATH,2,365.9727,-6083.048,249.1279,0,0,1,0,100,0),
+(@PATH,3,348.9503,-6058.797,249.1279,0,0,1,0,100,0),
+(@PATH,4,334.4755,-6041.239,249.1279,0,0,1,0,100,0),
+(@PATH,5,306.4679,-6010.077,240.8223,0,0,1,0,100,0),
+(@PATH,6,305.4742,-6011.148,217.1557,0,0,1,0,100,0),
+(@PATH,7,308.0244,-6019.713,217.1557,0,0,1,0,100,0),
+(@PATH,8,331.7084,-6052.974,217.1557,0,0,1,0,100,0),
+(@PATH,9,350.9348,-6071.563,217.1557,0,0,1,0,100,0),
+(@PATH,10,371.7133,-6102.678,217.1557,0,0,1,0,100,0),
+(@PATH,11,389.949,-6113.926,217.1557,0,0,1,0,100,0);
```
Comment (ShinDarth) [inline data/sql/dbfix/2016_08_23_00.sql]: this file should go under [data/sql/updates/pending_db_world](https://github.com/azerothcore/azerothcore-wotlk/tree/master/data/sql/updates/pending_db_world)

**Comment by Yehonal** [comment]
@DantestyleXD should i've to merge this or we need to wait for "TODO" first?

**Comment by DantestyleXD** [comment]
With this fix, the quest can be done.
I can try to fix the aggro too.

**Comment by DantestyleXD** [comment]
Rev.5
Enemies doesn't aggro anymore BUT, for a blizzlike experience, there need to modify core script about NPCs controlled and do an except for creature 24783 summoned by player.

With this fix, enemies doesn't aggro player and the summoned creature because ,the core, considers the creature like the player.

**Pull request merged**